package.json

```diff
@@ -62,7 +62,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git://github.com/inspect-js/object-inspect.git"
+    "url": "https://github.com/inspect-js/object-inspect.git"
```

Comment:
git+ssh instead please

Reply:
Now changed to `ssh://` per https://github.com/inspect-js/has-symbols/pull/22#discussion_r2274665816